lib/json-schema/validator.rb

```diff
@@ -238,6 +242,46 @@ def validation_errors
       @errors
     end
 
+    def resolve_nested_references(schema, schema_portion, list)
```

Comment:
The only function which receives the same combination, a schema object and a hash describing some property (or the entire) of the schema is in [get_referenced_uri_and_schema](https://github.com/ruby-json-schema/json-schema/blob/master/lib/json-schema/attributes/ref.rb#L21) in ```RefAttribute``` where the hash, here ```schema_portion``` is called ```s```.
What do you think of ```schema_hash``` as name? Or ```schema_portion_hash``` as it can also be some nested sub-property of the schema?